gitlab/config.py

```diff
@@ -165,108 +165,102 @@ def _parse_config(self) -> None:
             # Value Error means the option exists but isn't a boolean.
             # Get as a string instead as it should then be a local path to a
             # CA bundle.
-            try:
-                self.ssl_verify = _config.get("global", "ssl_verify")
-            except Exception:  # pragma: no cover
-                pass
-        except Exception:
+            self.ssl_verify = _config.get("global", "ssl_verify")
+        except (configparser.NoOptionError, configparser.NoSectionError):
             pass
         try:
             self.ssl_verify = _config.getboolean(self.gitlab_id, "ssl_verify")
         except ValueError:
             # Value Error means the option exists but isn't a boolean.
             # Get as a string instead as it should then be a local path to a
             # CA bundle.
-            try:
-                self.ssl_verify = _config.get(self.gitlab_id, "ssl_verify")
-            except Exception:  # pragma: no cover
-                pass
-        except Exception:
+            self.ssl_verify = _config.get(self.gitlab_id, "ssl_verify")
+        except (configparser.NoOptionError, configparser.NoSectionError):
             pass
 
         try:
             self.timeout = _config.getint("global", "timeout")
-        except Exception:
+        except (configparser.NoOptionError, configparser.NoSectionError):
```

Comment:
We were probably silently catching a lot of `ValueError` and `TypeError` in these excepts so far, especially for `getboolean` and `getint`. Not sure if we should also handle that in here? Haven't tested with offending values yet :)